packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -178,6 +178,7 @@ class PlanarFreehandROITool extends ContourSegmentationBaseTool {
       configuration: {
         shadow: true,
         preventHandleOutsideImage: false,
+        contourProcessingModifiers: KeyboardBindings.Shift,
```

Comment:
please add a comment to each of these on what they are

Reply:
It is in the plural because it also accepts  a combination of modifiers

```javascript
enum KeyboardBindings {
  Shift = 16,
  Ctrl = 17,
  Alt = 18,
  Meta = 91,
  ShiftCtrl = 1617,
  ShiftAlt = 1618,
  ShiftMeta = 1691,
  CtrlAlt = 1718,
  CtrlMeta = 1791,
  AltMeta = 1891,
}
```